mogc_partnerships/views.py

```diff
@@ -103,9 +103,9 @@ class CohortOfferingListView(generics.ListAPIView):
 
     def get_serializer_context(self):
         context = super().get_serializer_context()
-        context["enrollments"] = self.request.user.enrollment_records.values_list(
-            "offering_id", flat=True
-        )
+        context["enrollments"] = self.request.user.enrollment_records.filter(
+            is_active=True
+        ).values_list("offering_id", flat=True)
```

Comment:
We can use the `active` method of our custom manager here.

```suggestion
        context["enrollments"] = self.request.user.enrollment_records.active().values_list(
            "offering_id", flat=True
        )
```